observations/obs_converters/SIF/SIF_to_obs_netcdf.rst

```diff
@@ -0,0 +1,200 @@
+| 1.0 = lesser        |
++---------------------+
+| "........."         |
++---------------------+
+| 50  = least quality |
++---------------------+
+
+The `input_qc_threshold` namelist value can be used to test whether or not lesser 
+quality observations improve the result or not.  Thus, all observations are included 
+and the exclusion of observations is left up to the user based upon the `input_qc_threshold`.
+
+The qc value assignment is such where anything not assigned a EVI Quality value of 
+'good' (00), is assigned a qc=50.  Values that recieve a EVI Quality of 'good' (00), 
+then are further sorted based upon the VI Usefulness parameter as follows:
+
++------------------------------------------------------+----------------------------------+
+| EVI Quality Usefulness Parameter                     | DART-compatible QC               |
++======+===============================================+====+=============================+
+| 0000 |  Highest quality                              | 0  | Highest Quality             |
++------+-----------------------------------------------+----+-----------------------------+
+| 0001 | Lower quality                                 | 1  | Lower Quality               |
++------+-----------------------------------------------+----+-----------------------------+
+| 0010 | Decreasing quality                            | 2  | Lower Quality               |
++------+-----------------------------------------------+----+-----------------------------+
+| 0100 | Decreasing quality                            | 3  | Lower Quality               |
++------+-----------------------------------------------+----+-----------------------------+
+| 1000 | Decreasing quality                            | 4  | Lower Quality               |
++------+-----------------------------------------------+----+-----------------------------+
+| 1001 | Decreasing quality                            | 5  | Lower Quality               |
++------+-----------------------------------------------+----+-----------------------------+
+| 1010 | Decreasing quality                            | 50 | Do not use                  |
++------+-----------------------------------------------+----+-----------------------------+
+| 1100 | Lowest     quality                            | 50 | Do not use                  |
++------+-----------------------------------------------+----+-----------------------------+
+| 1101 | Quality so low that it is not useful          | 50 | Do not use                  |
++------+-----------------------------------------------+----+-----------------------------+
+| 1110 | L1B data faulty                               | 50 | Do not use                  |
++------+-----------------------------------------------+----+-----------------------------+
+| 1111 | Not useful for any other reason/not processed | 50 | Do not use                  |
++------+-----------------------------------------------+----+-----------------------------+
```

Comment:
Not all of the DART QC codes assigned by the converter are present in the table.

Reply:
I added all the current and updated input QC values (e.g. 1-7 and 10-17, 50) to reflect the EVI QC within the revised table.